src/master/develop/queries-history.md

```diff
@@ -650,9 +740,9 @@ const cl = new ImmudbClient({ host: IMMUDB_HOST, port: IMMUDB_PORT });
 ```
 :::
 
-::: tab .Net
+::: tab .NET
 This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [.Net sdk github project](https://github.com/codenotary/immudb4dotnet/issues/new)
```

Comment:
I think this is actually supported?

Reply:
VerifiableSetReference is not added. I was too much inspired by Java, I missed that.  I can add it soon; if you think it's an important one then I can start adding it.

Reply:
added